source/module_esolver/esolver_fp.cpp

```diff
@@ -305,4 +306,51 @@ void ESolver_FP::print_rhofft(const Input_para& inp, std::ofstream& ofs)
     }
 }
 
+#ifdef __LCAO
+void ESolver_FP::init_orb(double& dr_uniform,
```

Comment:
Please don't add new functions in ESolver_FP

Reply:
init_orb has been removed from ESolver_FP and placed in esolver/init_orb.hpp.